library/pod-security-policy/capabilities/template.yaml

```diff
@@ -16,10 +16,12 @@ spec:
           properties:
             allowedCapabilities:
               type: array
+              description: "An allow-list of Linux capabilities."
               items:
                 type: string
             requiredDropCapabilities:
               type: array
+              description: "A disallow-list of Linux capabilities."
```

Comment:
This may not be how this works. IIRC pods allow you to specify capabilities that are explicitly dropped, but then they can be re-added.

This supports someone saying "drop all capabilities, but re-add the NET capability"

Reply:
I see what you mean.  This random [blog post](https://kubesec.io/basics/containers-securitycontext-capabilities-drop-index-all/#example) gives the same example.  This is pretty vague in the k8s docs, but I guess it's a domain that most users of the feature might already have knowledge about.

Reply:
I've updated these descriptions to better fit my new understanding.  PTAL

Reply:
Change LGTM!